### PR TITLE
Explain how to document a language track

### PR DESCRIPTION
**Pull request by kytrinyx**
This addresses confusion that is documented in #3312

### CODE REVIEW

**Comment by Insti** [comment]
Did this mean to stop at xpascal?

**Comment by kytrinyx** [comment]
I got hit by abuse rate limits. Had to wait a bit to complete the list.

File: docs/writing-track-documentation.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* `ABOUT.md` - short, enticing introduction to the language
+* `LEARNING.md` - where to get an introduction to the language if you're learning it for the first time
+* `RESOURCES.md` - references, tools, etc.
+
+Any images referenced in the markdown files need to live within the `docs/img` directory. If the directory doesn't exist, you can create it when you need it.
+
+Use the relative path to this directory when referring to the images in the markdown. E.g., if you have an image named `docs/img/dropdown.png`, then you would reference it with:
+
+```
+![](/docs/img/dropdown.png)
```
Comment (joelwallis) [inline docs/writing-track-documentation.md]: @kytrinyx you're using an absolute path over here, instead of a relative one. Try using `` instead.

Comment (kytrinyx) [inline docs/writing-track-documentation.md]: excellent catch, thank you!

**Pull request merged**